Dockerfile

```diff
@@ -10,6 +10,7 @@ ADD . /go/src/github.com/dutchcoders/transfer.sh
 WORKDIR /go/src/github.com/dutchcoders/transfer.sh
 
 ENV GO111MODULE=on
+ENV APP_PORT=80
```

Comment:
this will work for new helm template but actually break everything else, please, keep `8080` as listener port

Reply:
yes, indeed.  Its my mistake.  I forgot about it while pushing.  Will make sure it stays same as per transfersh's convention.